### PR TITLE
A few fixes from testing

### PR DESCRIPTION
**Pull request by zoecarver**
Fixes rdar://109838935&109834813&109837096&110071334&109838608&109832749 

Each commit describes what it fixes specifically.

### CODE REVIEW

**Comment by zoecarver** [comment]
@swift-ci please test

File: lib/ClangImporter/ImportDecl.cpp [hunk 1]
```diff
@@ -3139,7 +3139,9 @@ namespace {
           // instead of checking if they come from the `std` module.
           if (!d->getDeclName().isIdentifier())
             return false;
-          return d->getName() == "abs" || d->getName() == "div";
+          return d->getName() == "abs" || d->getName() == "div" ||
```
Comment (egorzhdan) [inline lib/ClangImporter/ImportDecl.cpp:3142]: There are more of such functions in libc++, I wonder if we should do an audit and make all of them unavailable in Swift. This is not urgent, just a thought for the future.

**Review by egorzhdan — APPROVED** [review]

File: lib/ClangImporter/ImportDecl.cpp (hunk 1 above)
Comment (ravikandhadai) [reply lib/ClangImporter/ImportDecl.cpp:3142]: Yes. If we can mark them unavailable in Swift in libc++, that sounds like a much better way to do this.

**Comment by zoecarver** [comment]
@swift-ci please test

File: lib/ClangImporter/ImportDecl.cpp (hunk 1 above)
Comment (zoecarver) [reply lib/ClangImporter/ImportDecl.cpp:3142]: Exit is another one

**Comment by zoecarver** [comment]
@swift-ci please test linux

**Comment by zoecarver** [comment]
@swift-ci please test

**Comment by zoecarver** [comment]
@swift-ci please smoke test

**Comment by zoecarver** [comment]
@swift-ci please smoke test

**Comment by zoecarver** [comment]
@swift-ci please smoke test

**Comment by zoecarver** [comment]
@swift-ci please smoke test

**Pull request closed without merging**